docs/CONTRIBUTION.md

```diff
@@ -11,3 +11,21 @@ We welcome contributions to the Session Creator project! Here's how you can help
     - Create a new branch with a descriptive name (e.g., feature/name).
     - Make your changes, following established coding style and TypeScript best practices.
     - Open a pull request with a clear description of your contribution.
+
+## Docker Setup
+
+We have integrated Docker to streamline the development and deployment process. Here&apos;s how you can use it:
```

Comment:
We have "also" integrated. We don't want `docker` to be the main mode of accessing our code. This can be an optional layer.